session_test.go

```diff
@@ -1784,3 +1783,5 @@ func TestMaxIncomingStreams(t *testing.T) {
 	_, err = str.Read([]byte{0})
 	require.NoError(t, err)
 }
+
+// TODO: add unit test
```

Comment:
? Is this something you want to do this in this PR or a note for some future work?